fjord/feedback/views.py

```diff
@@ -44,23 +46,39 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    if waffle.flag_is_active(request, 'thankyou'):
-        template = 'feedback/thanks.html'
-        try:
-            opinion_id = request.session.get('opinion_id')
-            user_opinion = Response.objects.get(id=opinion_id)
-        except Response.ObjectDoesNotExist:
-            pass
-        else:
-            word_count = len(user_opinion.description.split())
-            if (user_opinion.locale == 'en-US' and
-                    not user_opinion.happy and
-                    word_count >= 7):
-                template = 'feedback/thanks_sad.html'
-    else:
-        template = 'feedback/thanks.html'
+    suggestions = None
+    response = None
 
-    return render(request, template)
+    if waffle.flag_is_active(request, 'thankyou'):
+        response_id = None
+
+        # If the user is an analyzer/admin, then we let them specify
+        # the response_id via the querystring. This makes debugging
+        # the system easier.
+        if ((request.user.is_authenticated()
+             and request.user.has_perm('analytics.can_view_dashboard'))):
```

Comment:
So this code is goof-tastic. It's semantically wrong, but I've been using it across the codebase. Now I just copy and paste it so that it's easier to replace it all when I get around to fixing it.

The problem is that this perm has nothing to do with being in the analyzer's group. I'm conflating having this permission with being part of the group, but I really should have a finer set of permissions that I should be testing which allow for multiple groups.

Anyhow, I'm going to keep it this way to make it easier to fix later.